cli/commands/run/command.go

```diff
@@ -0,0 +1,63 @@
+// Package run contains the logic for interacting with OpenTofu/Terraform.
+package run
+
+import (
+	"strings"
+
+	"github.com/gruntwork-io/go-commons/collections"
+	"github.com/gruntwork-io/terragrunt/internal/cli"
+	"github.com/gruntwork-io/terragrunt/internal/errors"
+	"github.com/gruntwork-io/terragrunt/internal/experiment"
+	"github.com/gruntwork-io/terragrunt/options"
+	"github.com/gruntwork-io/terragrunt/tf"
+)
+
+const (
+	CommandName = "run"
+)
+
+func NewCommand(opts *options.TerragruntOptions) *cli.Command {
+	return &cli.Command{
+		Name:        CommandName,
+		Usage:       "Run an OpenTofu/Terraform command. Shortcuts for common `run` commands are provided below.",
+		UsageText:   "terragrunt run [options] -- <tofu/terraform command>",
+		Description: "Run a command, passing arguments to an orchestrated tofu/terraform binary.\n\nThis is the explicit, and most flexible form of running an IaC command with Terragrunt. Shortcuts can be found in \"terragrunt --help\" for common use-cases.",
+		Examples: []string{
+			"# Run a plan\nterragrunt run -- plan\n# Shortcut:\n# terragrunt plan",
+			"# Run output with -json flag\nterragrunt run -- output -json\n# Shortcut:\n# terragrunt output -json",
+			"# Run a plan against a Stack of configurations in the current directory\nterragrunt run --all -- plan",
```

Comment:
```suggestion
```

We should be careful not to advertise functionality we don't support yet in docs. We can (and should) add this in once we support the flag.